web/client-api/src/main/java/io/deephaven/web/client/api/WorkerConnection.java

```diff
@@ -219,6 +222,7 @@ public WorkerConnection(QueryConnectable<?> info, Supplier<Promise<ConnectToken>
         inputTableServiceClient =
                 new InputTableServiceClient(info.getServerUrl(), JsPropertyMap.of("debug", debugGrpc));
         objectServiceClient = new ObjectServiceClient(info.getServerUrl(), JsPropertyMap.of("debug", debugGrpc));
+        partitionedTableServiceClient = new PartitionedTableServiceClient(info.getServerUrl(), JsPropertyMap.of("debug", debugGrpc));
```

Comment:
Probably, but the logging can be pretty noisy - the auth update logs would be every few seconds, and make it hard to tell what others were doing, so i usually enable only one at a time.

If you have a strong preference, I'll make these all have an empty map, and a specific "debug=true" object, rather than just default to false so any one boolean can be flipped.

Reply:
I don't have a strong preference.
I guess in JS I'd do something like:
```
const options = { debug: debugGrpc };
const inputTableClient = new ITSC(url, options);
const partitionedTableServiceClient = new PTSC(url, options);
... etc
```
Then if I wanted to flip just one:
```
const options = { debug: debugGrpc };
const inputTableClient = new ITSC(url, options);
const partitionedTableServiceClient = new PTSC(url, { ...options, debug: true });
... etc
```
Though I don't really know how best to translate that to Java land (`JsPropertyMap.of(options, "debug", debugGrpc)` or something?). And it's not that big a deal anyway.